tests/integration/crud/advanced/test_crud_advanced_example.py

```diff
@@ -0,0 +1,156 @@
+import os
+
+from jina import Document
+from jina.executors.indexers import BaseIndexer
+from jina.flow import Flow
+import pytest
+
+
+@pytest.fixture
+def config(tmpdir):
+    os.environ['JINA_CRUD_ADVANCED_WORKSPACE'] = str(tmpdir)
+    yield
+    del os.environ['JINA_CRUD_ADVANCED_WORKSPACE']
+
+
+def get_docs_to_index(contents):
+    for i, content in enumerate(contents):
+        document = Document()
+        document.id = str(f'{i}' * 16)
+        document.text = content
+        yield document
+
+
+def get_docs_to_delete(doc_id_to_chunk_ids):
+    for i, (doc_id, chunks) in enumerate(doc_id_to_chunk_ids.items()):
+        document = Document()
+        document.id = str(f'{i}' * 16)
+        for chunk in chunks:
+            document.chunks.append(chunk)
+        yield document
+
+
+def validate_index(tmpdir, validation_data):
+    assert len(validation_data) > 0
+    for index_file, expected_size in validation_data:
+        index = BaseIndexer.load(str(os.path.join(tmpdir, index_file)))
+        assert index.size == expected_size
+
+
+def test_crud_advanced_example(tmpdir, config, mocker, monkeypatch):
+    '''
+    This test indexes documents into an example flow and updates one document.
+    The update is implemented as delete & index.
+    '''
+    monkeypatch.setenv("RESTFUL", 'False')
```

Comment:
why dont we set to False in the Flow alrrady?

Reply:
thanks, I changed it